basiclibrary/src/main/java/basiclibrary/Averages.java

```diff
@@ -0,0 +1,17 @@
+package basiclibrary;
+
+public class Averages {
+
+    public static double arrayAverage(int[] array) {
+        double sum = 0;
+        double avg = 0;
+        for (int i=0; i < array.length; i++) {
+            sum += array[i];
+        }
+        avg = sum/(array.length);
+        System.out.println("sum: " + sum);
+        System.out.println("length: " + array.length);
+        System.out.println("avg: " + avg);
+        return avg;
```

Comment:
Instead of declaring `double avg = 0;` on ln 7, you can just do
`return sum/array.length;